test/integration/builder/inserts.js

```diff
@@ -1817,5 +1817,81 @@ module.exports = function (knex) {
           });
       });
     });
+
+    it('#4152 Should allow returns with inserts on tables with triggers', async function () {
```

Comment:
since there is lots of reusable setup here, I wonder if both should be moved to more focused file, e. g. 'triggers.js' and have shared `beforeEach`/`afterEach` statements.

Reply:
Okay. I can make it a new file and add it to the suite. I will then make the reused variables global for reuse for any additional tests we need. Thanks for all your help!

Reply:
Thank you for putting so much effort into fixing this!